lib/browser-tests/puppeteer-helpers.js

```diff
@@ -0,0 +1,67 @@
+import path from 'path';
+import fs from 'fs';
+import { version } from '../../package.json';
+
+export function delay(ms) {
+  return new Promise((resolve) => {
+    setTimeout(resolve, ms);
+  });
+}
+
+const pathToSinonjsBrowserPkgSrc = path.resolve(__dirname, '../../node_modules/sinon/pkg/sinon.js');
+
+export async function useFakeTimers(page, config = {}) {
+  await page.evaluateOnNewDocument(`
+    if (!window.clock) {
+      ${fs.readFileSync(pathToSinonjsBrowserPkgSrc, 'utf8')}
+      window.clock = sinon.useFakeTimers(${JSON.stringify(config)})
+    }
+  `);
+}
```

Comment:
we were using sinon before - but a part of it only, and to do that we had to compile the extracted part from the browser, I discovered that if we just use the whole sinon library, it's npm package has a compiled js file ready to use in the browser, so no more need for that extra build step in our tests